app/models/ability.rb

```diff
@@ -18,8 +18,10 @@ def initialize(user)
         can :manage, :all
       else
         can :read, Article
+        can :read, Post
         can :update, Article if user.has_role?(:manager, Article)
         can :update, Article if user.has_role?(:moderator, Article)
+        can :manage, Post, User_id: user.id 
```

Comment:
User_id? camel case?

Reply:
User_id is the field belongs to the Post table. so it has to be used the same as User_id.

Reply:
If I want to change this, I should change in the Post table migration.

Reply:
it should be **u**ser_id not **U**ser_id